lib/triangle_cases.rb

```diff
@@ -1,28 +1,34 @@
 class TriangleCase < OpenStruct
-  def name
+  def test_name
     initial = description.downcase
     replaced = initial.gsub(/(true|false)/, replacements)
     if initial.eql?(replaced) && !initial.include?(triangle)
       replaced = triangle + ' triangle ' + initial
     end
-    'test_%s' % replaced.gsub(/[ ,-]/, '_')
+    'test_%s' % replaced.gsub(/[, -]/, '_').squeeze('_')
```

Comment:
`gsub(/[, -]/, '_').squeeze('_')`
Ruby has a method that does this.
string.tr_s(', -','_')